app/services/authService.js

```diff
@@ -115,6 +117,42 @@ class AuthService {
     };
   }
 
+  async sendActivateEmail(userId, userIp) {
+    const oUser = await PortalUserModel.findById(userId).lean();
+    if (!oUser) {
+      throw new CustomError(STATUS_TYPE.PORTAL_USER_NOT_FOUND);
+    }
+    if (oUser.is_activated) {
```

Comment:
is_activated and siteName? two different naming rules followed!

Reply:
All the variables contain '_' means it is a field of a model in database, and without '_' like siteName is defined in the function.